jobs/openshift/cluster/integreatly/openshift-cluster-integreatly-install.yaml

```diff
@@ -12,14 +12,6 @@
           name: 'openshiftMasterUrl'
           default: ''
           description: '[REQUIRED] The public URL of the target OpenShift cluster'
-      - string:
```

Comment:
@mikenairn @odra Do we really need this as a parameter here? I've removed it because we don't really have a choice on what credentials are created on the cluster during provision.

Reply:
Probably not as long as the clusters we are testing against always use the same user/password. If we ever had to test on a cluster that wasn't created by tower, this could be different.